extra/release_tool.py

```diff
@@ -457,7 +457,9 @@ def get_docker_compose_data_from_json_list(json_list):
         for container, cont_info in json_elem["services"].items():
             full_image = cont_info.get("image")
             if full_image is None or (
-                "mendersoftware" not in full_image and "mender.io" not in full_image
+                "mendersoftware" not in full_image
+                and "mender.io" not in full_image
+                and "northern.tech/mender" not in full_image
             ):
```

Comment:
I honestly here do not understand why we add it, but I have little knowledge about release_tool code, so trust here.

Reply:
During rounds of testing I was setting the compose file to use the GitLab CI registry so that I could use the image from `devauth` pull request.

It is harmless and it might come in handy for the next time.

Reply:
okey, but we need a comment on it in here. please :)

Reply:
I am dropping this commit